source/_posts/2017-11-07-which-hass-to-use.markdown

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "Which installation should I do"
+description: "Should you use hass.io or VirtualEnv or hassbian?"
+date: 2017-11-07 10:00:00 +0000
+date_formatted: "November 07, 2017"
+author: Ryan McLean
+author_twitter: ryanm101
+comments: true
+categories: How-To
+---
+
+# Decisions, Decisions, Decisions...
+
+Ok you've made it this far you're interested in automating your home, but Home-assistant has you stuck there are 3 main ways to install it.. Which should you choose?
+
+*   Hass.io
+*   Python Virtual Environment
+*   Hassbian
+
+This brief article will hopefully give you a starting point to make your decision as all 3 are perfectly viable options. These are not prescriptive and hass.io for example can be installed on any linux server but if you are looking to do that, you don't need this article to tell you the options.
```

Comment:
Use `Hass.io`. Well, you can't install it on `any linux server`. There are [limitations](https://github.com/home-assistant/hassio-build/tree/master/install#supported-machine-types) and requirements.